packages/rest/README.md

```diff
@@ -64,3 +64,15 @@ const run = async (agent: Agent) => {
 // A Swagger (OpenAPI) definition is exposed on http://localhost:3000/docs
 run()
 ```
+
+### Webhooks
+
+We use webhooks as a method for the rest api to have the option to call the controller in case of an event.
+
+Usage: add a webhook url to the ServerConfig you pass to startServer()
+
+Current supported events are:
+
+- connections
+- credentials
```

Comment:
Could you elaborate a bit on the usage?

- How do you pass the `webhookUrl` (can be passed to both `startServer` and `setupServer`) -- Code examples are always nice!
- How is the `webhookUrl` used? Does it send everything to the webhook url, or does it append the topic to the url before sending? This is useful for people integrating with it.
- What is the payload sent in the body of the webhook? Is it a json object? If so, what is the content of the payload?

Reply:
E.g. look at the ACA-Py docs as an example: https://github.com/hyperledger/aries-cloudagent-python/blob/main/AdminAPI.md#administration-api-webhooks

I don't think we should add documentation on a per parameter level (as that becomes quickly outdated), but would be good to add some more context